spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -608,6 +630,27 @@ protected long getShutdownTimeout() {
 	}
 
 	/**
+	 * How often to emit {@link ListenerContainerIdleEvent}s in milliseconds.
+	 * @param idleEventInterval the interval.
+	 */
+	public void setIdleEventInterval(Long idleEventInterval) {
```

Comment:
I think we can require here an valid value already and really don't let to provide pointless `null`.
Therefore primitive `long`.
But, eh, minor, of course :smile: